src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -381,6 +383,135 @@ public Task GetMigratedServerUrls()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/databases/*/smuggler/migrate", "POST", AuthorizationStatus.ValidUser)]
```

Comment:
Admin operation?

Reply:
isn't it the same as an import?

Reply:
I think that it is the same as the other migration? `/databases/*/admin/smuggler/migrate`

BTW, since this is not exposed to the Client API, it would be better to change the
`/databases/*/admin/smuggler/migrate` -> `/databases/*/admin/smuggler/migrate/ravendb`

and this endpoint from `/databases/*/smuggler/migrate` -> `/databases/*/admin/smuggler/migrate`

Reply:
This *MUST* be an admin operation. 
We are allowing the user to specify an exec that we'll run. That means that we want to be pretty sure that only an authorized admin can do that, and not just any user.

Reply:
Also need to check that database restore from 3.x is admin only (which I think it is because this is creating the db)

Reply:
done

Reply:
DatabaseAdmin or Operator?

Reply:
changed to Operator as the smuggler/migrate/ravendb endpoint